config/locales/idv/en.yml

```
@@ -93,9 +93,9 @@ en:
         warning: Please check the information you entered and try again. Common mistakes
           are an incorrect Social Security number or ZIP Code.
       setup:
-        fail_html: 'Please call our contact center at <strong>%{contact_number}</strong>
-          to continue verifying your identity. You will need to provide them
-          with the error code <strong>%{support_code}</strong>.'
+        fail_date_html: Call our contact center by %{date} to continue verifying your identity.
+        fail_html: Call %{contact_number} and provide them with the error code
+          %{support_code}.
```

Since these don't contain HTML, we shouldn't need to exempt them from sanitization with the `_html` suffix.

```suggestion
        fail_date: Call our contact center by %{date} to continue verifying your identity.
        fail: Call %{contact_number} and provide them with the error code
          %{support_code}.
```